mysql/MySQLLexer.g4

```diff
@@ -32,7 +32,7 @@ channels { MYSQLCOMMENT, ERRORCHANNEL }
 SPACE:                               [ \t\r\n]+    -> channel(HIDDEN);
 SPEC_MYSQL_COMMENT:                  '/*!' .+? '*/' -> channel(MYSQLCOMMENT);
 COMMENT_INPUT:                       '/*' .*? '*/' -> channel(HIDDEN);
-LINE_COMMENT:                        ('-- ' | '#') ~[\r\n]* ('\r'? '\n' | EOF) -> channel(HIDDEN);
+LINE_COMMENT:                        (('-- ' | '#') ~[\r\n]* ('\r'? '\n' | EOF) | '--' ('\r'? '\n' | EOF)) -> channel(HIDDEN);
```

Comment:
The issue with the original version is the required trailing space. A comment existing only out of 2 dashes and a line break was not recognized and threw an error.